Makefile

```diff
@@ -63,12 +77,9 @@ build: test
 	@echo "Built $(BUILDS_DIR)/$($@FILE)"
 	rm -rf $(STAGING_DIR)
 
-_check-arn:  # Ensure that an ARN is set before we can deploy to it.
-ifndef ARN
-	$(error ARN is undefined; set to the appropriate Lambda ARN to deploy to. View with `make describe-stack`)
-endif
-
-deploy: build _check-arn
+update: _is_user_authenticated _check_stack_exists build
```

Comment:
Think I still prefer `deploy` as a target name

Reply:
Ok, no problem I can revert that. `Deploy` felt like a first action, instead of a later workflow, but it's not a big deal either way.